scenarios/kubernetes_build.py

```diff
@@ -31,6 +32,36 @@ def check(*cmd):
     subprocess.check_call(cmd)
 
 
+def get_gcs_path(args):
+    # xref https://github.com/kubernetes/release/blob/master/push-build.sh
```

Comment:
I can remove these, they are just to document the logic involved for this type of build's gcs path, I didn't want to modify push build since it's upstream and probably used for other things.